src/main/frontend/handler/events.cljs

```diff
@@ -408,6 +410,10 @@
     (js/console.error "instrument data-map should only contains [:type :payload]"))
   (posthog/capture type payload))
 
+(defmethod handle :capture-error [[_ {:keys [error payload]}]]
```

Comment:
any requirements on error and payload?
noticed that many error arg is (js/Error. "string").
does it have to be a js/Error?

Reply:
Yes, non-Error works too.
<img width="770" alt="CleanShot 2022-11-24 at 21 57 12@2x" src="https://user-images.githubusercontent.com/479169/203801840-63dfbdb0-a404-4b61-a82e-480b7f322486.png">
https://docs.sentry.io/platforms/javascript/usage/